packages/sanity/src/core/comments/__workshop__/CommentInlineHighlightDebugStory.tsx

```diff
@@ -17,8 +17,8 @@ import {useCallback, useEffect, useMemo, useRef, useState} from 'react'
 import {useCurrentUser} from '../../store'
 import {type CommentDocument} from '../types'
 import {
+  buildCommentRangeDecorations,
```

Comment:
Renamed this function to include the word Comments to make it a bit clearer what its for (internal export at the public level)